core/include/swatch/core/AbstractMetric.hpp

```diff
@@ -35,6 +36,19 @@ enum Status {
   kNonCritical,
   kDisabled,
 };
+
+const boost::unordered_map<std::string, Status> StringToStatus= {
```

Comment:
Ok. Can the initialisation be moved in cpp? There is no problem to initialise the map there.